### PR TITLE
Set up CI

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     long_description=open('README.md').read(),
     description='Articles management app for Django CMS',
     install_requires=[
-        'django-cms>=3.2',
+        'django-cms>=3.2,<3.4',
```
Comment (JsseL) [inline setup.py:19]: :+1: Since 3.4 seemed to have some plugin rendering related changes.

Comment (akx) [reply setup.py:19]: Yup. The plugin rendering tests didn't work whatsoever with 3.4.

**Comment by JsseL** [comment]
Looks good to me :ship:

**Comment by akx** [comment]
@JsseL Use the new GitHub code review tools to approve code ;)

**Pull request merged**